pgvectorscale/src/access_method/graph.rs

```diff
@@ -107,8 +119,14 @@ impl<QDM, PD> ListSearchResult<QDM, PD> {
     }
 
     /// Internal function
-    pub fn insert_neighbor(&mut self, n: ListSearchNeighbor<PD>) {
+    pub fn insert_neighbor(&mut self, mut n: ListSearchNeighbor<PD>) {
         self.stats.record_candidate();
+        if n.distance == 0.0 && self.tie_break_item_pointer.is_some() {
+            /* record the tie break if distance is 0 */
+            let tie_break_item_pointer = self.tie_break_item_pointer.unwrap();
+            let d = tie_break_item_pointer.ip_distance(n.index_pointer);
+            n.distance_tie_break = d;
+        }
```

Comment:
```suggestion
        if let Some(tie_break_item_pointer) = self.tie_break_item_pointer {
            /* record the tie break if distance is 0 */
            if n.distance == 0.0 {
                let d = tie_break_item_pointer.ip_distance(n.index_pointer);
                n.distance_tie_break = d;
            }
        }
```
NIT: this helps to remove the `unwrap`, gives the same result, and will help to avoid accidental unwraps in the future due to refactoring or the like.